### PR TITLE
[FEATURE] Reload weather every two minutes.

### PR DESCRIPTION
**Pull request by joelhawksley**
Add option for number of minutes the information is refreshed.
Forecast.io limits requests to 1000/day: a 2min interval = 720 calls/day

### CODE REVIEW

File: js/controller.js [hunk 1]
```diff
@@ -66,6 +66,32 @@
...
 
             var refreshMirrorData = function() {
+                CalendarService.getCalendarEvents().then(function(response) {
+                    $scope.calendar = CalendarService.getFutureEvents();
+                }, function(error) {
+                    console.log(error);
+                });
+
+                if ($scope.fitbitEnabled) {
+                    setTimeout(function() { refreshFitbitData(); }, 5000);
```
Comment (joelhawksley) [inline js/controller.js]: As an aside, I'd like to pull this out into its own function as well.

Comment (evancohen) [inline js/controller.js]: Yeah. Really each of the mirror components should be directives and self modify their scope and state.

File: config.example.js [hunk 2]
```diff
@@ -34,7 +34,8 @@ var config = {
     // forecast.io
     forecast : {
         key : "", // Your forecast.io api key
-        units : "auto" // See forecast.io documentation if you are getting the wrong units
+        units : "auto", // See forecast.io documentation if you are getting the wrong units
+        reload_interval : 2, // Number of minutes the information is refreshed. Forecast.io limits requests to 1000/day: a 2min interval = 720 calls/day
```
Comment (evancohen) [inline config.example.js]: To be consistent this really should be `reloadInterval`. It seems that RSS also has a different config property for reloading called `refreshInterval`. I don't really care which one we go with, I'd just like to be consistent.

Comment (joelhawksley) [inline config.example.js]: This is consistent with the `traffic` config, which is also `reload_interval`. Which would you prefer?

Comment (joelhawksley) [inline config.example.js]: (I'd be happy to give the config stack a style pass in a separate PR)

Comment (evancohen) [inline config.example.js]: Fine by me!

**Pull request merged**